share/instruments/instruments.xml

```diff
@@ -10946,6 +10946,28 @@
                   <longName>Double Bass</longName>
                   <shortName>Db.</shortName>
             </Instrument>
+            <Instrument id="octobass">
+                  <longName>Octobass</longName>
+                  <shortName>Otb.</shortName>
+                  <description>Octobass</description>
+                  <musicXMLid>strings.octobass</musicXMLid>
```

Comment:
The fact that MusicXML knows this instrument shows that we should have it too ;-)